sources/packages/backend/apps/api/src/route-controllers/application/models/application.dto.ts

```diff
@@ -163,6 +168,7 @@ export class ApplicationProgressDetailsAPIOutDTO {
   appealStatus?: StudentAppealStatus;
   scholasticStandingChangeType?: StudentScholasticStandingChangeType;
   applicationOfferingChangeRequestStatus?: ApplicationOfferingChangeRequestStatus;
+  assessmentTriggerType: AssessmentTriggerType;
```

Comment:
TriggerType should be optional as Submitted and Draft applications will not have it.